usdf/storage.rst

```diff
@@ -22,7 +22,7 @@ Science data under /sdf/group/rubin/:
  - datasets/
  - lsstdata/offline/ (still in prep)
  - repo/
- - shared/data/{test,validation}_data - daily updated checkouts of git lfs repos for use in testing code
+ - shared/data/test_data - daily updated checkouts of git lfs repos for use in testing code (includes the ap_verify CI datasets)
```

Comment:
You need to add another line for `validation_data`  if you're going to make this just `test_data`.

Reply:
We're getting rid of those datasets.